manifests/bucketeer/charts/event-persister-evaluation-events-dwh/templates/deployment.yaml

```diff
@@ -0,0 +1,163 @@
+              value: "{{ .Values.env.subscription }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PORT
+              value: "{{ .Values.env.port }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_METRICS_PORT
+              value: "{{ .Values.env.metricsPort }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_LOG_LEVEL
+              value: "{{ .Values.env.logLevel }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_MAX_MPS
+              value: "{{ .Values.env.maxMps }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_NUM_WORKERS
+              value: "{{ .Values.env.numWorkers }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_FLUSH_SIZE
+              value: "{{ .Values.env.flushSize }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_FLUSH_INTERVAL
+              value: "{{ .Values.env.flushInterval }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_FLUSH_TIMEOUT
+              value: "{{ .Values.env.flushTimeout }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PULLER_NUM_GOROUTINES
+              value: "{{ .Values.env.pullerNumGoroutines }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PULLER_MAX_OUTSTANDING_MESSAGES
+              value: "{{ .Values.env.pullerMaxOutstandingMessages }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_PULLER_MAX_OUTSTANDING_BYTES
+              value: "{{ .Values.env.pullerMaxOutstandingBytes }}"
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_CERT
+              value: /usr/local/certs/service/tls.crt
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_KEY
+              value: /usr/local/certs/service/tls.key
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_SERVICE_TOKEN
+              value: /usr/local/service-token/token
+            - name: BUCKETEER_EVENT_PERSISTER_DWH_BIGQUERY_DATA_SET
+              value: "{{ .Values.env.bigQueryDataSet }}"
+          volumeMounts:
+            - name: service-cert-secret
+              mountPath: /usr/local/certs/service
+              readOnly: true
+            - name: service-token-secret
+              mountPath: /usr/local/service-token
+              readOnly: true
+          ports:
+            - name: service
+              containerPort: {{ .Values.env.port }}
+            - name: metrics
+              containerPort: {{ .Values.env.metricsPort }}
+          livenessProbe:
+            initialDelaySeconds: {{ .Values.health.initialDelaySeconds }}
+            periodSeconds: {{ .Values.health.periodSeconds }}
+            failureThreshold: {{ .Values.health.failureThreshold }}
+            httpGet:
+              path: /health
+              port: service
+              scheme: HTTPS
+          readinessProbe:
+            initialDelaySeconds: {{ .Values.health.initialDelaySeconds }}
+            httpGet:
+              path: /health
+              port: service
+              scheme: HTTPS
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
```

Comment:
Please use `nindent` instead.
Same for the other places.

E.g.
https://github.com/bucketeer-io/bucketeer/pull/154/files#diff-9dda85b788b55f6d80b4593f70270c815fd495cdb8fc2e82192cc955da9c9959R25